src/main/kotlin/dk/cachet/carp/webservices/protocol/dto/ProtocolOverview.kt

```diff
@@ -0,0 +1,29 @@
+package dk.cachet.carp.webservices.protocol.dto
+
+import dk.cachet.carp.protocols.application.StudyProtocolSnapshot
+import java.time.Instant
+
+data class LatestProtocolOverview
```

Comment:
isnt this a dto as well? just to be consistent with the namings

Reply:
okey